core/controllers/reader_test.py

```diff
@@ -103,6 +106,59 @@ def test_published_explorations_are_visible_to_logged_in_users(self):
             expect_errors=True)
         self.assertEqual(response.status_int, 200)
 
+class ClassifyHandlerTest(test_utils.GenericTestBase):
+    """Test the hander for classification"""
+
+    def setUp(self):
+        super(ClassifyHandlerTest,self).setUp()
+
+    def test_classification_handler(self):
```

Comment:
Umm I think you need to make to cases - (1) to test that a correct answer is being classified correctly and the output answer group is correct, (2) to test that the output is default answer group given an incorrect answer. So, you can create two functions like - test_classification_handler_with_correct_answer and test_classification_handler_with_incorrect_answer . You can argue that this should be a part of test cases written for classifier services but I don't see that it would create any major difference.